cypress.config.js

```diff
@@ -1,10 +1,20 @@
-const { defineConfig } = require("cypress");
+const { defineConfig } = require("cypress")
 
 module.exports = defineConfig({
   e2e: {
-    baseUrl: 'http://localhost/Skosmos',
+    // You also can run like this: npx cypress run --config "baseUrl=http://localhost/skosmos"
+    baseUrl: 'http://localhost/skosmos',
```

Comment:
If you change this setting then running Cypress doesn't work for developers whose Skosmos installation is at /Skosmos instead of /skosmos. I hope we can eventually find a better way to run Cypress tests in a standardized setting, but meanwhile I would suggest not changing the baseUrl. Like the comment above says, it can always be overridden on the command line. Or you could rename your Skosmos installation to /Skosmos so it matches what others use.

Reply:
I suggest not changing the baseUrl but you can decide